<?xml version="1.0"?>
<launch>
  <group ns="secondary_robot">
    <node pkg="eurobot" name="stm_node" type="stm_node.py" respawn="true" output="screen"/>
    <node pkg="urg_node" name="lidar_node" type="urg_node" args="_ip_address:=192.168.0.10" respawn="true" output="screen">
      <param name="frame_id" value="/secondary_robot_laser"/>
    </node>
  </group>
</launch>
